sdk/eventhub/azure-eventhubs/HISTORY.md

```diff
@@ -4,6 +4,10 @@
 
 **Breaking changes**
 
+- Removed class `EventPostition`.
+    - The `initial_event_position` parameter of the `receive` method accepts offset(`str`), sequence number(`int`), datetime (`datetime.datetime`) or `dict` of these types.
+    - The `initial_event_position_inclusive` parameter of the `receive` method accepts `bool` or `dict` indicating whether the given event position is inclusive or not.
```

Comment:
I think based on the conversation with @johanste we were removing this "inclusive" parameter and potentially support the inclusive behaviour later via tuples or callables in the existing `initial_event_position` parameter.

Reply:
confirmed with Johan, taking the `initial_event_position_inclusive`.